tests/fixtures.py

```diff
@@ -30,45 +30,75 @@ def idle_policy(alice, bob):
-def enacted_policy(idle_policy):
+def idle_federated_policy(alice, bob):
+    """
+    Creates a Policy, in a manner typical of how Alice might do it, with a unique uri (soon to be "label" - see #183)
+    """
+    n = int(constants.NUMBER_OF_URSULAS_IN_NETWORK)
+    random_label = b'label://' + os.urandom(32)
+    policy = alice.create_policy(bob, label=random_label, m=3, n=n, federated=True)
+    return policy
+
+
+@pytest.fixture(scope="module")
+def enacted_blockchain_policy(idle_blockchain_policy, ursulas):
+    # Alice has a policy in mind and knows of enough qualifies Ursulas; she crafts an offer for them.
+    deposit = constants.NON_PAYMENT(b"0000000")
+    contract_end_datetime = maya.now() + datetime.timedelta(days=5)
+    network_middleware = MockRestMiddleware()
+
+    idle_blockchain_policy.make_arrangements(network_middleware, deposit=deposit, expiration=contract_end_datetime,
+                                             ursulas=list(ursulas))
+    idle_blockchain_policy.enact(network_middleware)  # REST call happens here, as does population of TreasureMap.
+
+    return idle_blockchain_policy
+
+
+@pytest.fixture(scope="module")
+def enacted_federated_policy(idle_federated_policy, ursulas):
     # Alice has a policy in mind and knows of enough qualifies Ursulas; she crafts an offer for them.
     deposit = constants.NON_PAYMENT(b"0000000")
     contract_end_datetime = maya.now() + datetime.timedelta(days=5)
     network_middleware = MockRestMiddleware()
 
-    idle_policy.make_arrangements(network_middleware, deposit=deposit, expiration=contract_end_datetime)
-    idle_policy.enact(network_middleware)  # REST call happens here, as does population of TreasureMap.
+    idle_federated_policy.make_arrangements(network_middleware,
+                                            deposit=deposit,
+                                            expiration=contract_end_datetime,
+                                            ursulas=ursulas)
+    idle_federated_policy.enact(network_middleware)  # REST call happens here, as does population of TreasureMap.
 
-    return idle_policy
+    return idle_federated_policy
 
 
 @pytest.fixture(scope="module")
-def alice(mining_ursulas, three_agents):
+def alice(ursulas, three_agents):
     token_agent, miner_agent, policy_agent = three_agents
     etherbase, alice, bob, *everyone_else = token_agent.blockchain.interface.w3.eth.accounts
 
     alice = Alice(network_middleware=MockRestMiddleware(),
                   policy_agent=policy_agent,
                   ether_address=alice,
-                  known_nodes=mining_ursulas,
-                  federated_only=True)
-    alice.recruit = lambda *args, **kwargs: [u.ether_address for u in mining_ursulas]
+                  known_nodes=ursulas,
+                  federated_only=True,
+                  abort_on_learning_error=True)
+    alice.recruit = lambda *args, **kwargs: [u._ether_address for u in ursulas]
```

Comment:
woah.. what?

Reply:
This just mocks out recruit so that Alice can proceed through the rest of the test.  There's almost certainly a better way to do this, but this is pretty good for now.